stdlib/public/TensorFlow/Tensor.swift

```diff
@@ -357,7 +358,7 @@ public extension Tensor {
   @differentiable(vjp: _vjpInit(repeating:shape:)
                   where Scalar : TensorFlowFloatingPoint)
   init(repeating repeatedValue: Scalar, shape: TensorShape) {
-    self = Raw.fill(dims: Tensor<Int32>(shape.dimensions),
+    self = Raw.fill(dims: Tensor<Int32>(shape.dimensions.map(Int32.init)),
```

Comment:
Have you tried changing swift-bindings so that they use `Int64` as the index type? The `index_type` attribute in the definition of `Fill` supports `int64`.

Reply:
Here's `Raw.fill`:
```swift
@inlinable @inline(__always)
public static func fill<T: TensorFlowScalar, IndexType: BinaryInteger & TensorFlowScalar>(
  dims: Tensor<IndexType>,
  value: Tensor<T>
) -> Tensor<T> {
  let ret: TensorHandle<T> = #tfop("Fill",
    dims,
    value,
    T$dtype: T.tensorFlowDataType,
    index_type$dtype: IndexType.tensorFlowDataType)
  return Tensor(handle: ret)
}
```

Are you suggesting to remove the `IndexType` generic parameter, to change `dims` to type `Tensor<Int64>`, and to pass `Int64.self` as `index_type$dtype`?

I would prefer that changes involving `swift-bindings` be handled in a follow-up.

Reply:
Since it's already generic, there's no need to change bindings.

Reply:
Could you please clarify what code changes you would like?

Reply:
I meant if TF supports `Int64` we should use `Int64` instead of doing truncation all the time. Here, you can `map(Int64.init)`.